taskcluster/ci/build/kind.yml

```diff
@@ -104,6 +104,18 @@ jobs:
         treeherder:
             symbol: nightly(B)
 
+    android-test:
```

Comment:
If we keep the new build type, `android-test` as a name conflicts with the name `androidTest`, which is used for unit/integration tests on device (it's an Android default name). I'm not sure what's a better name – I kind of just want it to say, "nightly" :)

Reply:
I agree, the name isn't good - I was thinking of calling it `opt` or `shippable`. I thought of calling it nightly as well but as I mentioned above, the term nightly should be reserved for things that are build once per day.